### PR TITLE
expression/aggregation: migrate test-infra to testify for agg_to_pb_test.go

### LINKED ISSUE #27905: migrate test-infra to testify for `expression/aggregation/agg_to_pb_test.go`
This is a subtask of #27904

### PR DESCRIPTION
**Pull request by penthaapatel**
Signed-off-by: Penthaa Patel <penthaapatel@gmail.com>

### What problem does this PR solve?

Issue Number: close #27905 

### What is changed and how it works?

What's Changed:

- migrate test infra to testify for `expression/aggregation/agg_to_pb_test.go`

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by tisonkun** [comment]
@penthaapatel I'm reviewing this PR now.

Please be patient if your patch is not applied as fast as you'd like (open source developers are all volunteers, often doing the development in their spare time) or a developer asks you to make changes to the patch. If you do not receive any feedback in a reasonable amount of time (say, a week or two), feel free to ping the reviewers again or post a "RFR" ("Request for Review") on our [developer discussion forum](https://internals.tidb.io/).

One or two reviewers in request is reasonable, ping more reviewers won't work expect create noise and bother people.

File: expression/aggregation/agg_to_pb_test.go [hunk 1]
```diff
@@ -52,17 +42,16 @@ type testEvaluatorSuite struct {
 	ctx sessionctx.Context
 }
 
-func (s *testEvaluatorSuite) SetUpSuite(c *C) {
+func createtestEvaluatorSuite(t *testing.T) (s *testEvaluatorSuite) {
+	s = new(testEvaluatorSuite)
 	s.Parser = parser.New()
 	s.ctx = mock.NewContext()
+	return
```
Comment (tisonkun) [inline expression/aggregation/agg_to_pb_test.go:49]: This suite is redundant also - you can just inline the setup lines.

**Comment by penthaapatel** [comment]
> @penthaapatel I'm reviewing this PR now.
> 
> Please be patient if your patch is not applied as fast as you'd like (open source developers are all volunteers, often doing the development in their spare time) or a developer asks you to make changes to the patch. If you do not receive any feedback in a reasonable amount of time (say, a week or two), feel free to ping the reviewers again or post a "RFR" ("Request for Review") on our [developer discussion forum](https://internals.tidb.io/).
> 
> One or two reviewers in request is reasonable, ping more reviewers won't work expect create noise and bother people.

@tisonkun Sure, will keep that in mind.

File: expression/aggregation/agg_to_pb_test.go (hunk 1 above)
Comment (penthaapatel) [reply expression/aggregation/agg_to_pb_test.go]: Fixed - 73c9037

File: expression/aggregation/agg_to_pb_test.go [hunk 2]
```diff
@@ -19,50 +19,25 @@ import (
...
-	s.Parser = parser.New()
-	s.ctx = mock.NewContext()
-}
-
-func (s *testEvaluatorSuite) TearDownSuite(c *C) {
-}
-
-func (s *testEvaluatorSuite) TestAggFunc2Pb(c *C) {
+func TestAggFunc2Pb(t *testing.T) {
+	ctx := mock.NewContext()
```
Comment (tisonkun) [inline expression/aggregation/agg_to_pb_test.go:39]:
```suggestion
func TestAggFunc2Pb(t *testing.T) {
	t.Parallel()
	ctx := mock.NewContext()
```

I guess this test can be run in parallel. Please confirm, by running locally and check whether it touches some global states.

Comment (penthaapatel) [reply expression/aggregation/agg_to_pb_test.go]: Yes, it works. No error occurs. - 46d7723

**Review by morgo — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
@morgo could you help on merging this PR?

**Pull request merged**